app.rb

```diff
@@ -0,0 +1,169 @@
+require './person'
+require './book'
+require './classroom'
+require './nameable'
+require './rental'
+require './student'
+require './teacher'
+# require './main'
+
+class App
+  def initialize()
+    @my_books = []
+    @my_rentals = []
+    @people = []
+  end
+
+  def show_menu
+    puts ' Welcome to school library'
+    puts ''
+    puts ' please choose an option by entering a number'
+    puts ' 1 - List all books'
+    puts ' 2 - List all people'
+    puts ' 3 - Create a person'
+    puts ' 4 - Create a book'
+    puts ' 5 - Create a rental'
+    puts ' 6 - List all rentals for a given person id.'
+    puts ' 7 - Exit'
```

Comment:
- [ ] Greate work 👍🏼 all the functionality of the app is working as expected and the code is distributed nicely in the main and app files, however, The app should not exit until the user selects `7`, as for now it can be closed if you click enter two times, kindly fix this bug so your app is completed